### PR TITLE
add health/ready endpoints

### PR DESCRIPTION
**Pull request by karlhungus**
#### Problem

I wanted to have a k8s pod for running my cloud watch exporter, but no readiness/health endpoint exists

#### Solution

Add one. These seem a little bit silly (they both just respond if the server is running, and don't say anything about the status of the connection to AWS (I took inspiration from prometheus: https://github.com/prometheus/prometheus/blob/2bd510a63e48ac6bf4971d62199bdb1045c93f1a/web/web.go#L297-L304) 

This also addresses: https://github.com/prometheus/cloudwatch_exporter/issues/32, which was closed but... @brian-brazil

### CODE REVIEW

File: src/main/java/io/prometheus/cloudwatch/HealthServlet.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class HealthServlet extends HttpServlet {
+
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        resp.setContentType("text/html");
+        resp.getWriter().print("OK");
```
Comment (brian-brazil) [inline src/main/java/io/prometheus/cloudwatch/HealthServlet.java]: This isn't valid HTML.

Comment (karlhungus) [reply src/main/java/io/prometheus/cloudwatch/HealthServlet.java]: Fair, i took inspiration from the prometheus health endpoint, which I don't believe is valid html either... would:

```html
<html>ok</html>
```

be sufficient?

Comment (brian-brazil) [reply src/main/java/io/prometheus/cloudwatch/HealthServlet.java]: Switching to text/plain is simplest.

Comment (karlhungus) [reply src/main/java/io/prometheus/cloudwatch/HealthServlet.java]: I guess to be html valid it'd have to be:
```html
<!doctype html>
<html lang=en>
<head>
<meta charset=utf-8>
<title>cloudwatch exporter</title>
</head>
<body>ok
</body>
</html>
```

Comment (karlhungus) [reply src/main/java/io/prometheus/cloudwatch/HealthServlet.java]: Thanks, i'm not a smart man :)

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**